android/res/values/strings.xml

```diff
@@ -612,9 +612,9 @@
 	<string name="privacy_policy">Privacy policy</string>
 	<string name="terms_of_use">Terms of use</string>
 	<string name="button_layer_traffic">Traffic</string>
-	<string name="button_layer_subway">Subway</string>
+	<string name="button_layer_subway">Metro</string>
```

Comment:
As I wrote before, these strings are used everywhere including in the US. So the proper fix should be to create `values-en-rUS` folder (if it's correct) and regenerate strings. And tune/fix Twine if necesary.